src/app/swap/[[...path]]/path-resolver.ts

```diff
@@ -33,13 +40,36 @@ export class PathResolver {
 
     let inputToken = this.resolveToken(symbols.inputToken ?? ETH.symbol)
     let outputToken = this.resolveToken(
-      symbols.outputToken ?? defaultIndex.symbol
+      symbols.outputToken ?? defaultIndex.symbol,
     )
+
+    if (
+      inputToken.symbol === Bitcoin2xFlexibleLeverageIndex.symbol &&
+      outputToken.symbol === IndexCoopBitcoin2xIndex.symbol
+    ) {
+      return {
+        isMinting: false,
+        inputToken: Bitcoin2xFlexibleLeverageIndex,
+        outputToken: IndexCoopBitcoin2xIndex,
+      }
+    }
+
+    if (
+      inputToken.symbol === Ethereum2xFlexibleLeverageIndex.symbol &&
+      outputToken.symbol === IndexCoopEthereum2xIndex.symbol
+    ) {
+      return {
+        isMinting: false,
+        inputToken: Ethereum2xFlexibleLeverageIndex,
+        outputToken: IndexCoopEthereum2xIndex,
+      }
+    }
```

Comment:
Quickest way right now to solve this, as otherwise the path resolving won't accept two indices as out and input token.